alerts/jvm/heap-memory-usage.json

```diff
@@ -0,0 +1,26 @@
+{
+    "displayName": "JVM - Heap Memory Usage Near Max",
+    "documentation": {
+        "content": "Alert for JVM heap memory usage being near maximum. When the JVM reaches its max, depending on the JVM configuration, this could lead to Out of Memory errors.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "JVM - Heap Memory Near Max",
+            "conditionMonitoringQueryLanguage": {
+                "duration": "1m",
```

Comment:
condition_monitoring_query_language.duration' (type.googleapis.com/google.protobuf.Duration), Field 'duration', Illegal duration format; duration must end with 's'

However, I don't think this field must be non-zero.

Reply:
@chanseokoh do you happen to know if there is a good way to validate all the protobuf declarations of these rather than hitting an account that already has the descriptors set up? 

Just looking to save your time so I don't have to bug you to validate!

Reply:
I am simply copy-pasting a json into the "Try this method" tool at https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.alertPolicies/create to call the create API against my test project. For me, I was able to make multiple calls with the same json successfully without deleting existing ones. (I just clean them later via Cloud Console.)

![image](https://user-images.githubusercontent.com/10523105/181377068-81e12836-1b34-457a-b3bc-b7101cb0b970.png)

![image](https://user-images.githubusercontent.com/10523105/181376930-73e1f634-ef38-4e69-ba65-28f3f4668901.png)

Reply:
BTW, what I noticed is that the `conditionThreshold.duration` must not be zero only when `evaluation_missing_data` is set. In this case, it's `conditionMonitoringQueryLanguage.duration` and not affected, so you can leave it to 0s if that's right.